WhaleSpotting/Repositories/UserRepo.cs

```diff
@@ -65,4 +66,18 @@ public User GetByUsername(string username)
             throw new ArgumentOutOfRangeException($"No user with username {username} found in the database", ex);
         }
     }
+    public User CreateNewUser(CreateUserRequest newUser)
+    {
+        var insertResponse = context.Users.Add(new User
+        {
+            Username = newUser.Username,
+            Password = newUser.Password,
+            ProfileImageUrl = newUser.ProfileImageUrl,
+            UserBio = newUser.UserBio,
+            UserType = (UserType)newUser.UserType,
+        });
+        context.SaveChanges();
+        return insertResponse.Entity;
+    }
```

Comment:
Sorry I think this was my confusion, as we already have the `CreateUser` function we can just edit the existing one

Reply:
Fixed